src/test/scala/com/cognite/sdk/scala/common/StringDataPointsResourceBehaviors.scala

```diff
@@ -156,16 +156,16 @@ trait StringDataPointsResourceBehaviors extends Matchers with OptionValues with
 
     it should "support support query by externalId when ignoreUnknownIds=true" in {
       val doesNotExist = CogniteExternalId(s"does-not-exist-${UUID.randomUUID.toString}")
-      dataPoints.getLatestStringDataPoints(Seq(doesNotExist), ignoreUnknownIds = true) shouldBe empty
-      dataPoints.getLatestDataPoints(Seq(doesNotExist), ignoreUnknownIds = true) shouldBe empty
+      dataPoints.getLatestStringDataPoints(Seq(doesNotExist),  true, Instant.now()) shouldBe empty
+      dataPoints.getLatestDataPoints(Seq(doesNotExist),  true, Instant.now()) shouldBe empty
       dataPoints.query(Seq(doesNotExist), Instant.EPOCH, Instant.now, ignoreUnknownIds = true) shouldBe empty
       dataPoints.queryStrings(Seq(doesNotExist), Instant.EPOCH, Instant.now, ignoreUnknownIds = true) shouldBe empty
     }
 
     it should "support support query by internal id when ignoreUnknownIds=true" in {
       val doesNotExist = CogniteInternalId(9007199254740991L)
-      dataPoints.getLatestStringDataPoints(Seq(doesNotExist), ignoreUnknownIds = true) shouldBe empty
-      dataPoints.getLatestDataPoints(Seq(doesNotExist), ignoreUnknownIds = true) shouldBe empty
+      dataPoints.getLatestStringDataPoints(Seq(doesNotExist),  true, Instant.now()) shouldBe empty
+      dataPoints.getLatestDataPoints(Seq(doesNotExist), true, Instant.now()) shouldBe empty
```

Comment:
```suggestion
      dataPoints.getLatestStringDataPoints(Seq(doesNotExist), true, Instant.now()) shouldBe empty
      dataPoints.getLatestDataPoints(Seq(doesNotExist), true, Instant.now()) shouldBe empty
```

Reply:
done in https://github.com/cognitedata/cognite-sdk-scala/pull/400/commits/d7aa30f9ef52a211d344fe8eb618e2f4d1fd1352